web/src/views/DandisetLandingView/DownloadDialog.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <v-menu
+    offset-y
+    :close-on-content-click="false"
+    left
+    min-width="500"
+    max-width="500"
+  >
+    <template v-slot:activator="{ on }">
+      <slot
+        name="activator"
+        :on="on"
+      />
+    </template>
+    <v-card>
+      <v-card-title>
+        Download full dandiset
+        <v-tooltip right>
+          <template v-slot:activator="{ on }">
+            <v-btn
+              href="https://www.dandiarchive.org/handbook/10_using_dandi/#downloading-from-dandi"
+              target="_blank"
+              rel="noopener"
+              icon
+            >
+              <v-icon
+                color="primary"
+                v-on="on"
+              >
+                mdi-help-circle
```

Comment:
Can you move the help ? to the far right side of the widget with Help text first, so e.g. `Help (?)`